ui/src/app/settings/views/Users/UsersList/UsersList.js

```diff
@@ -149,7 +166,37 @@ const Users = ({ initialCount = 20 }) => {
           defaultSortDirection="ascending"
           expanding={true}
           headers={[
-            { content: "Username", sortKey: "username" },
+            {
+              className: "p-table-multi-header",
+              content: (
+                <>
+                  <VanillaLink
+                    className={classNames("p-table-multi-header__link", {
+                      "is-active": displayUsername
+                    })}
+                    onClick={e => {
+                      e.preventDefault();
```

Comment:
In Vanilla you can get a link that looks like a button, but I wonder if it's a good idea to have a pattern for a button that looks like a link. I added `p-button--link` to RBAC because that way you could get all the default button behaviour (tabbing, focus, no need to prevent default) and eslint wouldn't complain about using links wrong.

Reply:
Thanks, I've added the --link pattern from rbac.